### ISSUE TITLE
More traditional sidebar option

### ISSUE THREAD

**Issue by TheMontyB** [BookStackApp/BookStack#1028]
Hi,
It'd be neat if we could get a traditional sidebar option - I've been looking at moving from Dokuwiki and this the key thing that I can't find nor make. Would help people find the pages they want a lot quicker without relying on search or having a good 100+ links on the main page...

Something like:
![image](https://user-images.githubusercontent.com/15889260/46011185-ffa22080-c0bc-11e8-979b-b59739e33d15.png)

Having standard text titles (or even books) with the chapters listed underneath would be more useful for getting users to where they need to be versus the current sidebar.

Is anything like this planned?

Thanks.

**Comment by lithium-ap** [comment]
Personally, I prefer the feed that is provided.

**Comment by TheMontyB** [comment]
> Personally, I prefer the feed that is provided.

Yeah, it might be best to have some sort of options (but I'd guess that'd be a lot more difficult).
Currently I have the sidebar style I'd like as the start page, but once you leave that, there's no quick jumping, which is a shame.

**Comment by edenpc** [comment]
An option for a more traditional sidebar would be nice as it would allow quicker navigation.

**Comment by raymanound** [comment]
I want this too. This is a nice feature to speed up the navigation and the suggestion from @moqmari seems good enough.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/1028
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
